File: src/main/java/nl/chimpgamer/networkmanager/extensions/discordbot/commands/mc/DiscordCommand.java
package nl.chimpgamer.networkmanager.extensions.discordbot.commands.mc;

import nl.chimpgamer.networkmanager.api.commands.NMBungeeCommand;
import nl.chimpgamer.networkmanager.api.sender.Sender;
import nl.chimpgamer.networkmanager.extensions.discordbot.DiscordBot;
import nl.chimpgamer.networkmanager.extensions.discordbot.utils.MCMessage;

import java.util.List;

public class DiscordCommand extends NMBungeeCommand {

    public DiscordCommand(DiscordBot discordBot, String name) {
        super(discordBot.getNetworkManager(), name, null);
        this.setPlayerOnly(true);
    }

    @Override
    public void onExecute(Sender sender, String[] strings) {
        sender.sendMessage(MCMessage.DISCORD_RESPONSE.getMessage());
    }

    @Override
    public List<String> onTabComplete(Sender sender, String[] strings) {
        return null;
    }
}

File: src/main/kotlin/nl/chimpgamer/networkmanager/extensions/discordbot/commands/mc/DiscordCommand.kt
package nl.chimpgamer.networkmanager.extensions.discordbot.commands.mc

import nl.chimpgamer.networkmanager.api.models.sender.Sender
import nl.chimpgamer.networkmanager.api.utils.commands.NMBungeeCommand
import nl.chimpgamer.networkmanager.extensions.discordbot.DiscordBot
import nl.chimpgamer.networkmanager.extensions.discordbot.configurations.MCMessage

class DiscordCommand(private val discordBot: DiscordBot, name: String) : NMBungeeCommand(discordBot.networkManager, name, null) {
    override fun onExecute(sender: Sender, args: Array<String>) {
        sender.sendMessage(discordBot.messages.getString(MCMessage.DISCORD_RESPONSE))
    }

    init {
        this.playerOnly = true
    }
}